### ISSUE TITLE
webpack

### ISSUE THREAD

**Issue by theskillwithin** [blueberryapps/react-bluekit#102]
feature request, ability to use with webpack?

or maybe even just ability to use with npm scripts.

**Comment by OscarBarrett** [comment]
Do you mean without having to use gulp? Because you can use the cli version of webpack/dev-server.

**Comment by theskillwithin** [comment]
yes!  webpack/dev-server ?

our backend is done with cakephp in vagrant, does this generate a static component i can use?

**Comment by ondrejbartas** [comment]
Hi in #99 we have support for cli commands so you dont need gulp. This task is generating index file which is then used as data for passing it to bluekit react component which you can add whenewer you want.

**Comment by theskillwithin** [comment]
thanks for the help! I still think I am a little lost tho, i am attempting it like this:

```
➜  v2 git:(release/mojo-v2) ✗ node_modules/react-bluekit/bin/bluekit.js --baseDir ./ --paths ./components
BlueKit generated components index to file: /Users/aupeterson/Projects/ansible-playbooks/marketplace/git-repos/marketplace/Apps/v2/componentsIndex.js
➜  v2 git:(release/mojo-v2) ✗ gss
 M package.json
?? componentsIndex.js
```
```
/* eslint-disable */
// Generated by component library do not modify

function dispatchEvent(data) {
  if (typeof window !== 'undefined') {
    const BluekitEvent = require('react-bluekit/lib/libraries/BluekitEvent');
    document.dispatchEvent(new BluekitEvent('functionTriggered', data));
  } else
    console.log('Bluekit received function triggered', data)
}

export default {
}

```

**Comment by theskillwithin** [comment]
https://gist.github.com/theskillwithin/758f7ed4933696148eef9b87aac2c704

**Comment by imtoo** [comment]
Hi, we merged [v0.4.1](https://github.com/blueberryapps/react-bluekit/releases/tag/0.4.1) into master I hope it's gonna help.

Btw. this gist is not accessible, it throws 404.
